test/unistd/links.c

```diff
@@ -17,39 +17,70 @@
 #include <emscripten.h>
 #endif
 
-void setup() {
-  int rtn = mkdir("working", 0777);
-  assert(rtn == 0);
-#if defined(__EMSCRIPTEN__) && defined(NODEFS)
-  EM_ASM(FS.mount(NODEFS, { root: '.' }, 'working'));
-#endif
-  rtn = chdir("working");
+void makedir(char *dir) {
+  int rtn = mkdir(dir, 0777);
   assert(rtn == 0);
-  symlink("../test/../there!", "link");
-  int fd = open("file", O_RDWR | O_CREAT, 0777);
+}
+
+void makefile(char *file, char *content) {
```

Comment:
Use `const char *` for both args here and args below.

Reply:
On it!